hieradata/cluster/antu.yaml

```diff
@@ -0,0 +1,35 @@
+---
+lookup_options:
+  network::interfaces_hash:
+    merge:
+      strategy: "deep"
+  yum::managed_repos:
```

Comment:
Why is this unique?

Reply:
As otherwise  conflicting yum repo names would attempt to be merged.  I'd rather get one out of 2 repos rather than a total trainwreck.

Reply:
(although I really wish a collision would be fatal...)